client/functions/basebuilding/fn_place_object.sqf

```diff
@@ -185,14 +185,43 @@ para_l_placing_click_handler = (findDisplay 46) displayAddEventHandler ["MouseBu
 	};
 }];
 
-// Adding key handler for rotation by 45 degrees (R key) 
+// Adding key handler for rotation by 45 degrees (R key) -Legend
 para_l_placing_keyrotate_handler = (findDisplay 46) displayAddEventHandler ["KeyDown", { 
 	params ["_displayorcontrol", "_keyCode"]; 
 	if (!scriptDone para_l_placing_script && _keyCode in [0x13]) then { // 0x13 is the 'R' key
 		para_l_placing_final_rotation = para_l_placing_final_rotation + 45; 
 	}; 
 }];
 
+// Update the event handler for the mouse scroll wheel, only works whilst holding ALT, probably due to "if (freeLook) exitWith {};" -FightKnight
```

Comment:
What dis?

Reply:
Same thing here, did not mean to put this in pull request

Reply:
are you using the github desktop application? or cli? i can show you some cli magic if you're that way inclined?

Reply:
Github desktop, yeah. I have the cli but just havent used it much.